classes/core/APIRouter.php

```diff
@@ -3,8 +3,8 @@
 /**
  * @file classes/core/APIRouter.php
  *
- * Copyright (c) 2014-2021 Simon Fraser University
- * Copyright (c) 2000-2021 John Willinsky
+ * Copyright (c) 2023 Simon Fraser University
+ * Copyright (c) 2023 John Willinsky
```

Comment:
(I'm not sure it's at all necessary, but our general practice is to extend the date when editing classes -- so these should be changed to `2014-2023` and `2000-2023`, not just `2023`.)